extension/manifest.json

```diff
@@ -3,44 +3,18 @@
     "gecko": {
       "id": "china-newtab@mozillaonline.com",
       "strict_min_version": "91.0",
-      "update_url": "https://addons.firefox.com.cn/chinaedition/addons/updates.json?reqVersion=%REQ_VERSION%&id=%ITEM_ID%&version=%ITEM_VERSION%&maxAppVersion=%ITEM_MAXAPPVERSION%&status=%ITEM_STATUS%&appID=%APP_ID%&appVersion=%APP_VERSION%&appOS=%APP_OS%&appABI=%APP_ABI%&locale=%APP_LOCALE%&currentAppVersion=%CURRENT_APP_VERSION%&updateType=%UPDATE_TYPE%&compatMode=%COMPATIBILITY_MODE%"
+      "update_url": "https://archive.mozilla.org/pub/cn_pack/addons.json"
     }
   },
-  "background": {
-    "scripts": [
-      "newtab/background.js"
-    ]
-  },
-  "chrome_url_overrides": {
-    "newtab": "newtab/newtab.html"
-  },
+  "background": {},
```

Comment:
When an extension does not need a background script, drop it for clarity.

```suggestion
```

P.S. We accept empty objects in `background` since Firefox 140 - https://bugzilla.mozilla.org/show_bug.cgi?id=1956847 . Before that, we refused to load the extension.